andela/odin-ah-frontend
# #160452446 In-app notification

#### What does this PR do?
 Add real-time in-app notification functionality
#### Description of Task to be completed?
-> Adds reducer and actions for fetching in-app notifications
-> Adds reducer and actions for updating  notifications
-> integrates socket.io for real-time notification
-> Adds a drop-down for displaying notifications
-> Adds test for notification reducers
->  Adds test for notification actions

#### How should this be manually tested?

```
git checkout staging 
```
```
 git pull
```
```
git checkout ft-160452446-view-notification
```
```
npm install
```
Register Author's Haven and login and you will see notification icon on the navbar
#### Any background context you want to provide?
Nil
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/160452446

#### Screenshots (if appropriate)
<img width="596" alt="a" src="https://user-images.githubusercontent.com/29021978/49148205-09d3db80-f307-11e8-9efc-cfaff28584ef.png">

#### Questions:
Nil
